src/components/MapView/MapView.css

```diff
@@ -14,3 +14,12 @@
   height: 100%;
   width: 100%;
 }
+
+.popup-container .mapboxgl-popup-content {
+  background-color: rgba(0, 0, 0, 0.7);
+}
+
+.popup-table {
+  font-size: 14;
```

Comment:
Please:

1. Define a new var in index.css: `--medium-font: 12px`
2. Update App.css top use the var on line 27: `font-size: var(--medium-font)`
3. Update your above code to use the var

Reply:
Done